api/src/index.ts

```diff
@@ -2,17 +2,24 @@ import "dotenv/config";
 import express, { Express } from "express";
 import { migrator } from "./database/config/databaseClient";
 import { redisClient } from "./database/config/redisClient";
-import cors from 'cors';
+import cors from "cors";
+import helment from "helmet";
 import assetsRouter from "./routes/AssetRouter";
 import locationsRouter from "./routes/LocationRouter";
 import settingsRouter from "./routes/SettingsRouter";
 import authRouter from "./routes/AuthRouter";
+import { rateLimterMiddleware } from "./middlewares/requestRateLimiter";
+import ErrorController from "./controllers/ErrorController";
 
 const app: Express = express();
 const port = process.env.API_PORT;
 
-app.use(cors())
+app.use(cors());
 app.use(express.json());
+app.use(ErrorController.errorHandler);
```

Comment:
M?: Would this not need to be after the `app.use` statements defining the routes, so that it is the `next` middleware in the stack from the routes.

Reply:
Commit: 5407ff0a58f6e756de0c895c64d6b62931bcea9a

Reply:
Commit: 6f141ce8314e21fb9bb6c950cb0fdcdd6d20d841